ganalytics.php

```diff
@@ -493,13 +493,20 @@ public function wrapProduct($product, $extras, $index = 0, $full = false)
 				'url' => isset($product['link']) ? $product['link'] : '',
 				'price' => number_format($product['price'], '2')
 			);
-
-			$ga_product = array_map('urlencode', $ga_product);
+			$ga_product = function_exists('json_encode')
+					? json_encode($ga_product)
+					: array_map(array($this, 'escapeJs'), $ga_product);
 		}
 
 		return $ga_product;
 	}
 
+	protected function escapeJs($text)
+	{
+		include_once SMARTY_PLUGINS_DIR . 'modifier.escape.php';
```

Comment:
even if a construct `include_once` should be used in the parenthesised form for uniformity IMHO